centrallix-lib/Makefile.in

```diff
@@ -183,6 +184,10 @@ valtest:	$(TESTPROGS)
 .PHONY: clean
 clean:
 	@rm -f src/*.o src/*.lo $(TESTPROGS) ${FINAL_TARGETS} 2>/dev/null
+	@rm -f *.gcov */*.gcov
+	@rm -f *.gcda */*.gcda
+	@rm -f *.gcno */*.gcno
+	@rm lcov.info
```

Comment:
Recommend a separate phony 'cov-clean' target to rm the coverage datafiles, so these metrics can be easily reset without recompiling.  'clean' would depend on 'cov-clean'.